src/odemis/gui/cont/streams.py

```diff
@@ -661,16 +661,25 @@ def _link_resolution(self):
                 self._binva = getattr(self.stream, fn)
                 break
         else:
-            logging.warning("Stream has resolution VA but no binning/scale, "
-                            "so it will not be updated.")
-            return
+            if hasattr(self.stream, "spectrum_binning"):
```

Comment:
Also check that "angular_binning" is present.

Reply:
is it needed? If the "spectrum_binning" is present, the "angular binning" should be too.

Reply:
Right now, indeed it's redundant. However, it would make the code more future proof if we extend some stream with a "spectrum_binning" without adding "angular binning". I also find it less "weird" when reading the code, because otherwise asymmetrical (and it's not explained why).